lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -74,7 +74,7 @@ public class InitialiseIpvSessionHandler
     private static final String REQUEST_EMAIL_ADDRESS_KEY = "email_address";
     private static final String REQUEST_VTR_KEY = "vtr";
     private static final List<Vot> HMRC_PROFILES_BY_STRENGTH = List.of(Vot.PCL250, Vot.PCL200);
-
+    private static final String EVIDENCE = "evidence";
```

Comment:
We have a class, `VerifiableCredentialConstants`, that contains a bunch of constants like this we can use. `evidence` is already in there so probably worth pulling that it, rather than redefining here.